[2023-08-13T14:17:18.868+0000] {taskinstance.py:1103} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: emrPipeline.s3_file_check manual__2023-08-13T14:16:33+00:00 [queued]>
[2023-08-13T14:17:18.876+0000] {taskinstance.py:1103} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: emrPipeline.s3_file_check manual__2023-08-13T14:16:33+00:00 [queued]>
[2023-08-13T14:17:18.876+0000] {taskinstance.py:1308} INFO - Starting attempt 1 of 31
[2023-08-13T14:17:18.898+0000] {taskinstance.py:1327} INFO - Executing <Task(S3KeySensor): s3_file_check> on 2023-08-13 14:16:33+00:00
[2023-08-13T14:17:18.905+0000] {standard_task_runner.py:57} INFO - Started process 6410 to run task
[2023-08-13T14:17:18.909+0000] {standard_task_runner.py:84} INFO - Running: ['***', 'tasks', 'run', 'emrPipeline', 's3_file_check', 'manual__2023-08-13T14:16:33+00:00', '--job-id', '910', '--raw', '--subdir', 'DAGS_FOLDER/emr.py', '--cfg-path', '/tmp/tmpiaq16fl_']
[2023-08-13T14:17:18.912+0000] {standard_task_runner.py:85} INFO - Job 910: Subtask s3_file_check
[2023-08-13T14:17:18.938+0000] {warnings.py:109} WARNING - /home/***/.local/lib/python3.9/site-packages/***/settings.py:195: DeprecationWarning: The sql_alchemy_conn option in [core] has been moved to the sql_alchemy_conn option in [database] - the old setting has been used, but please update your config.
  SQL_ALCHEMY_CONN = conf.get("database", "SQL_ALCHEMY_CONN")

[2023-08-13T14:17:18.994+0000] {task_command.py:410} INFO - Running <TaskInstance: emrPipeline.s3_file_check manual__2023-08-13T14:16:33+00:00 [running]> on host 484e6be077e8
[2023-08-13T14:17:19.171+0000] {taskinstance.py:1545} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='emrPipeline' AIRFLOW_CTX_TASK_ID='s3_file_check' AIRFLOW_CTX_EXECUTION_DATE='2023-08-13T14:16:33+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2023-08-13T14:16:33+00:00'
[2023-08-13T14:17:19.173+0000] {s3.py:103} INFO - Poking for key : s3://myawsbucket19080/hiveStep/raw/tripdata.csv
[2023-08-13T14:17:19.183+0000] {base.py:73} INFO - Using connection ID 'aws_connection' for task execution.
[2023-08-13T14:17:19.183+0000] {connection_wrapper.py:340} INFO - AWS Connection (conn_id='aws_connection', conn_type='aws') credentials retrieved from login and password.
[2023-08-13T14:17:19.870+0000] {base.py:255} INFO - Success criteria met. Exiting.
[2023-08-13T14:17:19.880+0000] {taskinstance.py:1345} INFO - Marking task as SUCCESS. dag_id=emrPipeline, task_id=s3_file_check, execution_date=20230813T141633, start_date=20230813T141718, end_date=20230813T141719
[2023-08-13T14:17:19.913+0000] {local_task_job_runner.py:225} INFO - Task exited with return code 0
[2023-08-13T14:17:19.946+0000] {taskinstance.py:2653} INFO - 1 downstream tasks scheduled from follow-on schedule check
